common/test/acceptance/pages/lms/progress.py

```diff
@@ -115,7 +115,7 @@ def _chapter_index(self, title):
         Return the CSS index of the chapter with `title`.
         Returns `None` if it cannot find such a chapter.
         """
-        chapter_css = '.chapters section .hd'
+        chapter_css = '.chapters>section h3'
```

Comment:
This one does matter. The results of `.chapters>section` are a strict subset of the results of `.chapters section`. Further, we're using this query to find a chapter index which gets passed to the query `section_css = '.chapters>section:nth-of-type({0}) .sections div .hd a'.format(chapter_index)` ([source](https://github.com/edx/edx-platform/blob/b8f257a0e1e02b98a25e797aa618dfbeb6eb773f/common/test/acceptance/pages/lms/progress.py#L137)).

The index found by the original (superset) query was not stable - I never nailed down what the cause of this was, but it varied from run-to-run due to timing, or test ordering maybe? The upshot was that we'd select `chapter_index` as an index of the superset (say, `section` number 4 out of 6), then try to use that index on the subset and get an error (Can you get me the 4th `section`? No, there are only 2 to choose from. KABOOM).